src/app/createCompany/styles.module.css

```
@@ -1,13 +1,17 @@
+.createCompanyWrapper {
+  @apply w-full h-screen bg-gray-300 md:flex flex-col justify-center items-center;
```

```suggestion
  @apply w-full h-screen bg-gray-300 md:flex md:flex-col md:justify-center md:items-center;
```

you don't have to put 'md:' on all changes?

thought about putting a shadow on the form? it can be good!

---

I think that put a shadow is a good idea

I didn't need to put md in all the styles because the rest of the styles are already responsive by default